recipe/bld.bat

```diff
@@ -6,10 +6,13 @@ set CXXFLAGS=
 cmake -G "NMake Makefiles" ^
       -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
       -D CMAKE_BUILD_TYPE=Release ^
+      -D OPENJPEG_INCLUDE_DIR=%LIBRARY_INC%\openjpeg-2.3 ^
```

Comment:
Probably wouldn't hurt to have `-D CMAKE_PREFIX_PATH=%LIBRARY_PREFIX%` in here too to stop it looking in the `mingw` install for stuf...